### DISCUSSION TITLE
ETH Denver Slide in security lesson

### DISCUSSION THREAD

**Discussion by TheWilsonGlobal** [Cyfrin/foundry-full-course-cu#2057] [General]
Could you please send URL of ETH Denver Slide in security lesson as below image 
![Screen Shot 2024-07-19 at 10 56 35](https://github.com/user-attachments/assets/b5c33e21-649a-4324-835a-4bb17d961126)

@PatrickAlphaC

**Comment by EngrPips** [comment]
Hello @TheWilsonGlobal, I just want to mention that Patrick will answer this as soon as they are available.

**Comment by PatrickAlphaC** [comment]
Here you go!

https://docs.google.com/presentation/d/1eQLELz6MYAWdZsb_wTnPfdE-R0dksSLh1YAWl7ti21M/edit?usp=sharing

**Reply by TheWilsonGlobal** [reply]
Thank you so much, Mr Patrick!!!
